src/components/state-details.js

```diff
@@ -230,6 +230,13 @@ export default function StateDetailsPage({ location, data }) {
   const currentSolarMWStr = numberToHumanString(currentSolarMW)
   const currentWindMWStr = numberToHumanString(currentWindMW)
 
+  // #### Number of Turbines and Football-fields of solar panels. ####
+  // https://www.usgs.gov/faqs/how-many-homes-can-average-wind-turbine-power
+  const averageTurbinePowerGenerated = 2.75
```

Comment:
We should clarify this and be more specific, I'd call it the `averageWindTurbineCapacityMw` and note that capacity is not the amount of power generated in a period of time, but the peak instantaneous power output.